### DISCUSSION TITLE
public keys

### DISCUSSION THREAD

**Discussion by MarkoSipovac** [MarkoSipovac/Tafe22S216#3] [General]
Hey guys just need your public keys to add them in

**Comment by ArthurWomacks** [comment]
Hi mate,

My public key: 

 AAAAB3NzaC1yc2EAAAADAQABAAABAQCHmgaeMK2ugnfMHTrjxPEo3CvYfGfQuSlY
ZkfpPL97KDVJEqUnzSxmtkcE8fPvEMBhbOBv+Mh8OUH95hmEZydsGsT2roFIFUQ5
/rwSF8mdjBrbUHkz+FwKxo9Aof7CuQ4NFyMDEcx6JJx23wJJA/5WkJ5VycNvr62a
GElaSRAwkCGrIkUporP5OJ8xCfSkOQYn3idm4zMz+AMJRjS6DWJTosOOaVNd0DIk
7MmstsIxw4bbz2txENQI6hLXSeK7cPEyAinyM0qhXUJJs1C8HO2aFFrH0RK/pagO
SWhDhXfDrYe61004x7ON1ULk+OCpkbRAok6JImCNFttQTGuJRsA1

**Reply by MarkoSipovac** [reply]
I don't think that's the correct one it should look something alone these line

SHA256:KeEAcrNTUC+UB5OFqqjb8cs6Ki7FnN6CtJd1NaZpfRY

**Reply by ArthurWomacks** [reply]
How did you get such short key?

Did you use PuTTYgen?

**Reply by MarkoSipovac** [reply]
I just did it exactly how the exercises showed us how. Its in the session 4 exercises document

**Comment by ArthurWomacks** [comment]
I follow the same instructions and always get a long key. According to Session 4 class exercise (section 11)  the key on the picture is long.

Did you use RSA key type?

**Reply by MarkoSipovac** [reply]
yeah im unsure how ours are different then. But i do know yours should start with "sha"

**Comment by ArthurWomacks** [comment]
Got it! 

SHA256:lkn56ub7TyuUG3MdT4mVU1W5Y3qs6e1xq58VbJMOD4o

**Reply by MarkoSipovac** [reply]
sweet i will add it now

**Reply by MarkoSipovac** [reply]
im getting this error when i try to add it. I have no idea whats going on or how to fix it
![error](https://user-images.githubusercontent.com/112137502/189787733-45d0e8a1-635e-4322-9369-523977b969c5.png)

**Comment by ArthurWomacks** [comment]
Try to use the long one and put space between ssh-rsa like here:
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCHmgaeMK2ugnfMHTrjxPEo3CvYfGfQuSlY
ZkfpPL97KDVJEqUnzSxmtkcE8fPvEMBhbOBv+Mh8OUH95hmEZydsGsT2roFIFUQ5
/rwSF8mdjBrbUHkz+FwKxo9Aof7CuQ4NFyMDEcx6JJx23wJJA/5WkJ5VycNvr62a
GElaSRAwkCGrIkUporP5OJ8xCfSkOQYn3idm4zMz+AMJRjS6DWJTosOOaVNd0DIk
7MmstsIxw4bbz2txENQI6hLXSeK7cPEyAinyM0qhXUJJs1C8HO2aFFrH0RK/pagO
SWhDhXfDrYe61004x7ON1ULk+OCpkbRAok6JImCNFttQTGuJRsA1

**Reply by MarkoSipovac** [reply]
I got a key already in use error. I might message KT Lau and ask him

**Comment by ArthurWomacks** [comment]
Ok

**Reply by MarkoSipovac** [reply]
ok so KTs recommendation is that you generate a new private/public key pair and then give me the public key and you use the private one

**Comment by ArthurWomacks** [comment]
New key: SHA256:fbm7zUbzNpFOfuAJu1Ao5/JOfp1/f5+FsXsSUXM8nWA

**Reply by MarkoSipovac** [reply]
thats the public key yeah?

**Reply by ArthurWomacks** [reply]
Yes

**Reply by MarkoSipovac** [reply]
its saying the same error as before the one that I sent a photo with

**Reply by ArthurWomacks** [reply]
Try this:
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCBVY9Fhoehk95Qdqy0yKxxbIwPRCuHcaTQ
EgwYjiLQfjsoNxQOyudC/WP3J6jXWM0QqmYnI5SuBbjSaLxUz4bjlIelKoVOgDPW
wJZX+dpVO5Chj69M2VjQ8HkBxKvUQwTgQU+OICh9A/P1f6y/yNUXCAKGoMfyuEu4
g2vxYZWAlf6UE3URsU1g9Gx3qFkwnyzwW7TiiG7fmCNHDm83UjaEFo+Kgvo0iDt7
64bPl1P5R81o7kzb1RccktOYYpzF1sPKBElFCflFYvI3RCDavfZ+NWirO6Dr2sao
NpY01fdieAsXXLveOtsibmUa6rWro+GO1R8PzaG2LPLo+WXyiv/H == rsa-key-20220913

**Reply by MarkoSipovac** [reply]
already in use again 😭 no idea what to do

**Reply by ArthurWomacks** [reply]
Try again

**Reply by ArthurWomacks** [reply]
This one
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCBVY9Fhoehk95Qdqy0yKxxbIwPRCuHcaTQ
EgwYjiLQfjsoNxQOyudC/WP3J6jXWM0QqmYnI5SuBbjSaLxUz4bjlIelKoVOgDPW
wJZX+dpVO5Chj69M2VjQ8HkBxKvUQwTgQU+OICh9A/P1f6y/yNUXCAKGoMfyuEu4
g2vxYZWAlf6UE3URsU1g9Gx3qFkwnyzwW7TiiG7fmCNHDm83UjaEFo+Kgvo0iDt7
64bPl1P5R81o7kzb1RccktOYYpzF1sPKBElFCflFYvI3RCDavfZ+NWirO6Dr2sao
NpY01fdieAsXXLveOtsibmUa6rWro+GO1R8PzaG2LPLo+WXyiv/H == rsa-key-20220913

**Reply by MarkoSipovac** [reply]
Success!!!!!!!!!

**Reply by ArthurWomacks** [reply]
Excellent!

**Comment by Kristian-HD** [comment]
this is my key

ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAQEAuX9Hi1FuClKDnzmpTQLurK5j5lT++4JyjAjftISdYQor05cpGe0uqhzMfTzFbjyQPDhjSB63RRBe2zAOMCU5LB7Or8dZCt5OombHZqd0WIy16ZLDU22U6vJr6SEXNkAj6HF7jggs6mWnmGnAdX/jIucyhELiFHznznJaObai32qdD0GZuPM2u7XIS20YBihUleN1Ipy90gctj9hn/kh2HeBh4eAEnLJKtMgU1QtHCr7KOmjPrNG/NPQthizN+pXv8pNwdy489BvDS5rUYq1xVMwY3YvT+UXUtMXhvYT1tjIcbC686r5N/lV66hneX7+UFgaas604P9QLKu3gBUdiRQ== rsa-key-20220901

**Reply by MarkoSipovac** [reply]
Same issue of already in use

**Reply by Kristian-HD** [reply]
try this one

ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAQEAmgr5GlPVhxN2OA0Cp4axPecYlbtPCGF0pdTHq/GDKPleImkoTp2DJ2cvI/WyK0shkiIIqyFn8q0BohF4RLOPhjrSTuzFexswKszCtEaFAK8bpU6xvZf9oddUouxz14Q5IF+Y+zIuFMPt0VXTFgmJIWAPn3Mpy3RG5/zJeIkkhLjzOpOR2NPMXEj/BoLiYKCbxer3QmuwbLWFORLeOT+HdVeqanUCtIfvY8bOB9iMBpk/P5KnFhJMnu+jsWjhS1yZfMwLUHu1Au4gDAvitC2phnWzQd4dSZXBfbkIGJKOCUKTAxrU7DNpcNbzdZHJKiInlm6jKqR4BoFS9D84g61UoQ== rsa-key-20220913

**Reply by MarkoSipovac** [reply]
worked!
